R/bs-theme-preview.R

```diff
@@ -352,10 +361,11 @@ bs_themer <- function(gfonts = TRUE, gfonts_update = FALSE) {
     }
 
     # Remember, theme at this point has been updated to reflect the current Bootswatch theme
-    changed_vals <- as.list(diff_css_values(
-      vals[sass_vars],
-      bs_get_variables(theme, names(vals[sass_vars]))
-    ))
+    theme_vals <- bs_get_variables(theme, names(vals[sass_vars]))
+    # Themer control for font-size-base is unitless
+    theme_vals[["font-size-base"]] <- sub("rem$", "", theme_vals[["font-size-base"]])
```

Comment:
I don't know if you want to make a helper method as it is _complicated_ and duplicated code from above...

```r
adjust_theme_vals <- function(theme_vals) {
  theme_vals[["font-size-base"]] <- sub("rem$", "", theme_vals[["font-size-base"]])
  theme_vals
}
theme_vals <- adjust_theme_vals(theme_vals)
```

I'm guessing there will be more situations where this will occur in the future